runtime/runtime/src/actions.rs

```diff
@@ -478,26 +497,16 @@ pub(crate) fn action_implicit_account_creation_transfer(
         // It holds because in the only calling site, we've checked the permissions before.
         AccountType::EthImplicitAccount => {
             if checked_feature!("stable", EthImplicitAccounts, current_protocol_version) {
-                // TODO(eth-implicit) Use real Wallet Contract.
-                let wallet_contract = wallet_contract_placeholder();
                 let storage_usage = fee_config.storage_usage_config.num_bytes_account
-                    + fee_config.storage_usage_config.num_extra_bytes_record
-                    + wallet_contract.code().len() as u64;
-
-                *account =
-                    Some(Account::new(transfer.deposit, 0, *wallet_contract.hash(), storage_usage));
-
-                // TODO(eth-implicit) Store a reference to the `Wallet Contract` instead of literally deploying it.
-                set_code(state_update, account_id.clone(), &wallet_contract);
```

Comment:
It was briefly discussed on Zulip that we could have a placeholder value (as opposed to real Wasm code) that is still set. The reason to do this would be to still signal to tooling like explorers that there is code deployed to these accounts (as opposed to accounts where there is no code at all), it's just that the deployed code is not explicitly specified in the state. It also could be a baby step towards a future generalized code sharing feature.

That said, I like the simplicity of simply storing nothing and using the convention that all Eth implicit accounts have an implicit contract also. Mostly this comment is here to kick off a discussion with the Runtime team about the design choice. Personally I am in favor of the option you have chosen here.

Reply:
Do you remember which topic the discussion happened?
I thought it is https://near.zulipchat.com/#narrow/stream/295306-pagoda.2Fcontract-runtime/topic/Eth-wallet.20support but I do not see it there.

Reply:
Yes, that's the one I was referring to. Though, again, I am happy with what is already implemented in this PR. It's only a question of if someone from Runtime teams thinks it is important to push for having some kind of code represented in the state.

Reply:
@Ekleog Could you take a look at this PR as someone from the Runtime team?

Reply:
I submitted this as a comment on near/NEPs#518 :)

And this brings me to this PR. Should we actually review it before near/NEPs#518 lands? This feels to me like doing the mistake we did near/NEPs#483 again, and we should not land any implementation before agreeing on the design.

(Also, I’d suggest adding near/NEPs#518 to the top post, as AFAIU this PR is supposed to be implementing it.)

Reply:
Following @Ekleog-NEAR suggestion [here](https://near.zulipchat.com/#narrow/stream/295306-pagoda.2Fcontract-runtime/topic/Eth-wallet.20support/near/403646004) I've changed it to deploy near[wallet contract hash]. ETH-implicit accounts will now store hash of these magic bytes.